<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDockWidget" name="camerasDockWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>400</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>378</width>
      <height>183</height>
     </size>
    </property>
    <property name="floating">
     <bool>false</bool>
    </property>
    <property name="allowedAreas">
     <set>Qt::AllDockWidgetAreas</set>
    </property>
    <property name="windowTitle">
     <string>Cameras</string>
    </property>
    <widget class="QWidget" name="camerasDockWidgetContents">
     <widget class="QTableWidget" name="camerasTableWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>360</width>
        <height>192</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dockWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>85</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="allowedAreas">
     <set>Qt::AllDockWidgetAreas</set>
    </property>
    <property name="windowTitle">
     <string>Settings</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_4">
     <widget class="QCheckBox" name="flashCheckBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Flash</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="preshootCheckBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>36</y>
        <width>78</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Preshoot</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="shootingDockWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>400</width>
      <height>110</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>95</height>
     </size>
    </property>
    <property name="windowTitle">
     <string>Shooting</string>
    </property>
    <widget class="QWidget" name="shootingDockWidgetContents">
     <widget class="ParameterSlider" name="tvSlider" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>382</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="ParameterSlider" name="avSlider" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>29</y>
        <width>382</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="ParameterSlider" name="isoSlider" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>49</y>
        <width>382</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="ParameterSlider" name="delaySlider" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>68</y>
        <width>382</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dockWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>400</width>
      <height>263</height>
     </rect>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="features">
     <set>QDockWidget::AllDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::AllDockWidgetAreas</set>
    </property>
    <property name="windowTitle">
     <string>Properties</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dockWidget_8">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="windowTitle">
     <string>Focus</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_8">
     <widget class="QCheckBox" name="manualFocusCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="focusSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>8</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReloadCameras"/>
   <addaction name="actionShoot"/>
   <addaction name="actionConfigureStaticProps"/>
   <addaction name="actionShutdownAll"/>
   <addaction name="actionDiagnose"/>
   <addaction name="actionSelectedCameraShoot"/>
   <addaction name="actionDownloadLastPhotos"/>
  </widget>
  <action name="actionReloadCameras">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload Camera List</string>
   </property>
   <property name="toolTip">
    <string>Update the list of cameras</string>
   </property>
  </action>
  <action name="actionShoot">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/camera-photo.png</normaloff>:/icons/camera-photo.png</iconset>
   </property>
   <property name="text">
    <string>Shoot</string>
   </property>
   <property name="toolTip">
    <string>Take one photo using each of the available cameras</string>
   </property>
  </action>
  <action name="actionDiagnose">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/document-export.png</normaloff>:/icons/document-export.png</iconset>
   </property>
   <property name="text">
    <string>Diagnose</string>
   </property>
  </action>
  <action name="actionConfigureStaticProps">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/applications-development.png</normaloff>:/icons/applications-development.png</iconset>
   </property>
   <property name="text">
    <string>Set up camera static parameters</string>
   </property>
   <property name="toolTip">
    <string>Set up camera static parameters</string>
   </property>
  </action>
  <action name="actionShutdownAll">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/system-shutdown.png</normaloff>:/icons/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Shutdown</string>
   </property>
   <property name="toolTip">
    <string>Shutdown all cameras</string>
   </property>
  </action>
  <action name="actionSelectedCameraShoot">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/selected-camera-photo.png</normaloff>:/icons/selected-camera-photo.png</iconset>
   </property>
   <property name="text">
    <string>SelectedCameraShoot</string>
   </property>
  </action>
  <action name="actionDownloadLastPhotos">
   <property name="icon">
    <iconset resource="photobooth.qrc">
     <normaloff>:/icons/download-last-photos.png</normaloff>:/icons/download-last-photos.png</iconset>
   </property>
   <property name="text">
    <string>DownloadLastPhotos</string>
   </property>
  </action>
  <zorder>shootingDockWidget</zorder>
  <zorder>dockWidget</zorder>
  <zorder>dockWidget_2</zorder>
  <zorder>dockWidget_5</zorder>
  <zorder>dockWidget_6</zorder>
  <zorder>dockWidget_7</zorder>
  <zorder>dockWidget_8</zorder>
  <zorder>dockWidget_8</zorder>
  <zorder>dockWidget_8</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParameterSlider</class>
   <extends>QWidget</extends>
   <header>parameterslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="photobooth.qrc"/>
 </resources>
 <connections/>
</ui>
